packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -4,171 +4,96 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import React, {useEffect, useState} from 'react'
-import {useQuery} from '@tanstack/react-query'
-
+import React from 'react'
 import {useApplicationExtensions} from '@salesforce/pwa-kit-extension-sdk/react'
 
-import HelloTS from '../components/hello-typescript'
-import HelloJS from '../components/hello-javascript'
-
-interface Props {
-    value: number
-}
-
 const style = `
-body {
-    background: linear-gradient(-45deg, #e73c7e, #23a6d5, #ee7752);
-    background-size: 400% 400%;
-    height: 100vh;
-}
-@keyframes gradient {
-    0% {
-        background-position: 0% 50%;
-    }
-    50% {
-        background-position: 100% 50%;
-    }
-    100% {
-        background-position: 0% 50%;
-    }
-}
-@keyframes fade {
-  0% { opacity: 0 }
-  100% { opacity: 1 }
-}
-.fade-in {
+.content {
     font-size: 18px;
-    opacity: 0;
-    animation: fade 1s ease-in-out;
-    animation-fill-mode: forwards;
-}
-.fade-in-0 { animation-delay: 0s}
-.fade-in-1 { animation-delay: 4s}
-.fade-in-2 { animation-delay: 8s}
-.fade-in-3 { animation-delay: 12s}
-.fade-in-4 { animation-delay: 16s}
-.fade-in-5 { animation-delay: 20s}
-body {
     font-family: "Helvetica", sans-serif;
-    font-weight: 300;
-    color: rgba(255,255,255,0.8);
-    color: chartreuse;
+}
+.content h1 {
+    font-size: 8em;
+    font-weight: 900;
+    letter-spacing: -0.05em;
+}
+.content p,
+.content ul {
+    margin: 1em 0;
 }
 .loading-screen {
-    mix-blend-mode: color-dodge;
     display: flex;
     flex-direction: row;
     flex-wrap: nowrap;
     justify-content: center;
     align-items: center;
     height: 100vh;
 }
-h1 {
-    font-size: 10em;
-    font-weight: 900;
-    letter-spacing: -0.05em;
-}
 .title {
     text-align: right;
 }
 .divider {
-    mix-blend-mode: lighten;
     width: 8px;
-    background-color: chartreuse;
-    height: 507px;
+    background-color: black;
+    height: 300px;
     margin-left: 5em;
     margin-right: 3em;
 }
+.button {
+    display: inline-block;
+    padding: 10px 20px;
+    font-size: 16px;
+    font-weight: bold;
+    color: white;
+    background-color: #0176D3;
+    text-align: center;
+    text-decoration: none;
+    border-radius: 5px;
+}
 `
 
-const Home = ({value}: Props) => {
-    const [counter, setCounter] = useState(0)
+const Home = () => {
```

Comment:
Is there any reason why we don't want to called this "Start" or "GettingStarted"? I personally like the later, but I would that the path is in alignment with the page name.. so /_pwa-kit/getting-started and GettingStarted or /_pwa-kit/start and Start

Reply:
Agreed with Ben on this